<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TankLevel</class>
 <widget class="QWidget" name="TankLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TankLevel</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>771</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QwtPlot" name="qwtPlot"/>
    </item>
    <item row="0" column="1">
     <widget class="QwtSlider" name="sliderAbsoluteLevel">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="scalePosition">
       <enum>QwtSlider::RightScale</enum>
      </property>
      <property name="handleSize">
       <size>
        <width>32</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QwtSlider" name="sliderPercentageLevel">
      <property name="minimumSize">
       <size>
        <width>36</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="scalePosition">
       <enum>QwtSlider::RightScale</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QwtTextLabel" name="labelAbsLevel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QwtTextLabel" name="labelPercentLevel">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>qwtPlot</zorder>
  <zorder>Slider_3</zorder>
  <zorder></zorder>
  <zorder>sliderPercentageLevel</zorder>
  <zorder>labelAbsLevel</zorder>
  <zorder>labelPercentLevel</zorder>
  <zorder>sliderAbsoluteLevel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
